R/as-data-frame.R

```diff
@@ -73,3 +73,45 @@ as.data.frame.CrunchDataFrame <- function (x, row.names = NULL, optional = FALSE
     names(out) <- names(x)
     return(structure(out, class="data.frame", row.names=c(NA, -nrow(x))))
 }
+
+
+#' as.data.frame method for VariableCatalog
+#'
+#' This method allows you to import the VariableCatalog into your R sessions in 
+#' order to facilliate interactive use. Modifying the dataframe produced by this
+#' method will not update the Crunch web app, for information about modifying 
+#' variables in the app see: \code{vingette("variables",
+#' package = "crunch")}  
+#' 
+#' @param x
+#' A VariableCatalog produced by \code{variables(ds)}.
+#' @param ... Additional arguments passed to \code{data.frame}
```

Comment:
This is stylistic, but I usually put the ... at the end (unless it's one of the functions where we use ... for the main arguments like `makeCaseVariable`)

Reply:
Is there a reason we're keeping the ... up at the beginning in the documentation? It's at the end in the function call.